.github/workflows/go.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         go get -v -t -d ./...
 
     - name: Build
-      run: go build -v ./...
```

Comment:
Not sure I understand what this change does and why make install is better.

Reply:
the go build command literally doesn't work.  Make install has lots of detailed options and works.